CHANGELOG.markdown

```diff
@@ -1,3 +1,7 @@
+0.6.2.1 [202y.mm.d]
+-------------------
+* Use `build-type: Simple`
```

Comment:
To be fair, it's a little more than just that. We also are removing a test suite in favor of a different approach, so that deserves to be mentioned as well.

Reply:
oops. Fixed.

Reply:
If you can draft a proper notice, I can add it to `comonad` and `lens` too.

Reply:
One possible way to phrase it would be:

```suggestion
* The `doctests` test suite has been removed in favor of using `cabal-docspec`. As a result, `distributive` no longer needs a custom `Setup` script, so its build type has been changed to `Simple`.
```

I'm not sure how discoverable the reference to `cabal-docspec` is, given that it's not currently on Hackage at the moment. Perhaps linking to its GitHub repo would suffice?

Reply:
I think we should say the `build-type: Simple` first. That is the important bit.

I don't think I'll distribute `cabal-docspec` (ditto `cabal-env` e.g.) via Hackage any time soon (I don't want to version its dependencies, or pretend they are useful for others).

Reply:
> I think we should say the `build-type: Simple` first. That is the important bit.

That's fine with me. But it is worth mentioning that a test suite has been removed, and I think it would be mysterious not to mention the reason why it was removed. I'm open to suggestions on the best way to phrase this, however.

Reply:
> The build-type of a package has been changed to `Simple`. To achieve that the `doctests` test suite has been removed in favor of using [`cabal-docspec`](https://github.com/phadej/cabal-extras/tree/master/cabal-docspec) to run doctests.

?

Reply:
Sounds good to me. Here's the same sentence, but with some minor touch-ups:

> The build-type has been changed from `Custom` to `Simple`. To achieve this, the `doctests` test suite has been removed in favor of using [`cabal-docspec`](https://github.com/phadej/cabal-extras/tree/master/cabal-docspec) to run the doctests.

Reply:
Good. Updated this, and also `comonad` and `lens` PRs.

Reply:
Awesome, thanks! I'll land these and make any necessary Hackage releases tomorrow morning.